package.json

```diff
@@ -115,5 +115,8 @@
   "volta": {
     "node": "16.13.2",
     "npm": "8.3.1"
+  },
+  "dependencies": {
```

Comment:
Any reason it's not with all the others deps in devDependencies ? I'm not sure about the consequences of it though